### ISSUE TITLE
Create Listing MetaMask signing Not Working

### ISSUE THREAD

**Issue by SidSarvesh** [nftdevs/NFTs-Upload-to-OpenSea#32]
Facing Continuous Error After Creating NFT. it's Not Able to Click the Create Listing Button and Sign with Metamask.

selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button type="submit" class="Blockreact__Block-sc-1xf18x6-0 Buttonreact__StyledButton-sc-glfma3-0 bhqEJb fzwDgL">...</button> is not clickable at point (239, 725). Other element would receive the click: <div data-testid="toasts" class="Blockreact__Block-sc-1xf18x6-0 Flexreact__Flex-sc-1twd32i-0 ToastsContainerreact__StyledContainer-sc-ihijgw-0 dBFmez jYqxGr kPAxwi"></div> (Session info: chrome=96.0.4664.110)

**Comment by mateeej00** [comment]
"SidSarvesh" is this the same issue?:
https://github.com/nftdevs/NFTs-Upload-to-OpenSea/issues/37

**Comment by SidSarvesh** [comment]
> "SidSarvesh" is this the same issue?: #37

yes. i faced same issue on my laptop. so i have changed my machine. 

i've decided to run this program on my  old desktop and it's working fine. sometimes it will threw the same error. we have to re run the program again to temporarily solve the issue. 
use Different monitor or different machine to cross check it. 
it's weird thing for me too. but it's works.

i've uploaded almost 3000+ NFTs with this.

**Comment by cloudmaking** [comment]
This may be of use...
https://github.com/cloudmaking/Auto_Chrome_2

**Comment by okanaira** [comment]
I get the following error on my laptop, don't have access to a different machine:

raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button type="submit" class="Blockreact__Block-sc-1xf18x6-0 Buttonreact__StyledButton-sc-glfma3-0 bhqEJb fzwDgL">...</button> is not clickable at point (239, 725). Other element would receive the click: <div data-testid="toasts" class="Blockreact__Block-sc-1xf18x6-0 Flexreact__Flex-sc-1twd32i-0 ToastsContainerreact__StyledContainer-sc-ihijgw-0 dBFmez jYqxGr kPAxwi"></div>
  (Session info: chrome=96.0.4664.110)
